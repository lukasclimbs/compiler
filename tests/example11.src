(+ (if (<= 3 5) 3 5) (- 21 78))) 7h
