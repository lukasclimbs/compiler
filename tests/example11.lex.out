Fatal error: exception Failure("Unexpected character found: h")
